docs/standard/serialization/snippets/system-text-json-how-to-5-0/csharp/HttpClientExtensionMethods.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+using System.Net.Http;
+using System.Net.Http.Json;
+using System.Threading.Tasks;
+
+namespace HttpClientExtensionMethods
+{
+    public class User
+    {
+        public int Id { get; set; }
+        public string Name { get; set; }
+        public string Username { get; set; }
+        public string Email { get; set; }
+    }
+    public class Program
+    {
+        public static async Task Main()
+        {
+            var client = new HttpClient();
+            client.BaseAddress = new Uri("https://jsonplaceholder.typicode.com");
```

Comment:
Do we have any Microsoft/dotnet provided dummy JSON producing endpoints? There was feedback previously about pointing to an external URL. FWIW I think this url is safe and we can keep using it if there's no other option.

Reply:
I looked for and could not find a Microsoft/dotnet provided dummy JSON producing endpoint.